src/curly.ml

```diff
@@ -234,27 +234,40 @@ let run prog args stdin_str =
   ; stderr = Buffer.contents err_buf
   }
 
-let run ?(exe="curl") ?(args=[]) req =
+let is_redirect_code status =
+  status <= 308 && status >= 300
+
+let run ?(exe="curl") ?(args=[]) ?(follow_redirects=true) req =
```

Comment:
Can you switch the default to `false` to avoid surprising existing users?

Reply:
Sure, I just pushed a fix.